client/cmd/validator.go

```diff
@@ -710,6 +732,35 @@ func unstakeOnBehalf(ctx context.Context, cfg unstakeConfig) error {
 }
 
 func unjail(ctx context.Context, cfg unjailConfig) error {
+	uncompressedValidatorPubKeyBytes, err := uncompressPrivateKey(cfg.PrivateKey)
+	if err != nil {
+		return errors.Wrap(err, "failed to get uncompressed pub key from private key")
+	}
+
+	result, err := prepareAndReadContract(ctx, &cfg.baseConfig, "fee")
+	if err != nil {
+		return err
+	}
+
+	var unjailFee *big.Int
+	err = cfg.ABI.UnpackIntoInterface(&unjailFee, "fee", result)
+	if err != nil {
+		return errors.Wrap(err, "failed to unpack unjailFee")
+	}
+
+	fmt.Printf("Unjail fee: %s\n", unjailFee.String())
+
+	_, err = prepareAndExecuteTransaction(ctx, &cfg.baseConfig, "unjailOnBehalf", unjailFee, uncompressedValidatorPubKeyBytes, []byte{})
```

Comment:
We should call unjail instead of unjailOnBehalf here

Reply:
Thanks for pointing out. It was missed. Applied in [591a551](https://github.com/piplabs/story/pull/369/commits/591a551140e52fb636c60c214e165fe68984f2df).